### ISSUE TITLE
exporting of Artifacts directly from an Artifact Cache does not work 

### ISSUE THREAD

**Issue by gregcaporaso** [qiime2/q2cli#350]

**Comment by Oddant1** [comment]
@lizgehret I went to assign myself and somehow didn't notice you were already assigned. Think I was just on auto pilot. Do you mind if I take this one?

**Comment by lizgehret** [comment]
@Oddant1 yes that's fine! Do you mind if I review your PR once it's ready?

**Comment by Oddant1** [comment]
@lizgehret sounds good to me
